# Name of the app to get details of
string name 
---
ExchangeApp app

